src/SprykerEco/Yves/Heidelpay/Form/InvoiceSecuredB2CSubForm.php

```diff
@@ -14,62 +14,51 @@
 use SprykerEco\Shared\Heidelpay\HeidelpayConfig;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
-class AbstractHeidelpaySubForm extends AbstractSubFormType implements SubFormInterface, SubFormProviderNameInterface
+class InvoiceSecuredB2CSubForm extends AbstractSubFormType implements SubFormInterface, SubFormProviderNameInterface
```

Comment:
I'm not sure that GitHub shows correct changes here but is it okay to get rid of AbstractHeidelpaySubForm which might be used on the project level?

Reply:
Abstract class are evil. They are not extendable and in my opinion abstract class has to have at least one abstract method. It just a clean up. It will be major.